### PR TITLE
Journalists can attach an image to their articles

### PR DESCRIPTION
**Pull request by Fraseer**
PT Link: https://www.pivotaltracker.com/story/show/180142611

## User Story
As a news site
In order to attract readers to an article
I would like to have the ability to add images to articles

## Changes Proposed
- add `cypress-file-upload`
- Put `selectedImage` into state
- create a to Base 64 module
- update form to take image 

## Screenshots
image

### CODE REVIEW

File: cypress/integration/userCanCreateArticle.feature.js [hunk 1]
```diff
@@ -12,15 +11,15 @@ describe("User can create an article", () => {
...
-  
-  it("is expected to find a page header and 6 input fields", () => {
+
+  it("is expected to find a page header and 7 input fields", () => {
     cy.get("[data-cy=page-header]").should("contain", "Create Article");
-    cy.get("[data-cy=create-article").children().should("have.length", 6);
+    cy.get("[data-cy=create-article").children().should("have.length", 7);
   });
 
   it("is expected input the following data", () => {
```
Comment (AugustEnhager) [inline cypress/integration/userCanCreateArticle.feature.js]: Maybe "is expected _to_ input the following data?

**Review by AugustEnhager — APPROVED** [review]

**Review by dorianbuck — APPROVED** [review]
LGTM

File: src/modules/apiHelper.js [hunk 2]
```diff
@@ -5,8 +5,8 @@ const Article = {
   async create(article) {
     article["article"]["author_ids"] = [];
     const params = article;
-    const headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"))
-    const response = await axios.post("/api/articles/", params, headers);
+    const headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+    const response = await axios.post("/api/articles/", { params, headers });
```
Comment (maxarvid) [inline src/modules/apiHelper.js]: Is there a reason we have to wrap the params and header arguments as an object?

Comment (Fraseer) [reply src/modules/apiHelper.js]: The authorization wasn't working on Netlify after the last merge, so I added {} to be inline with the end user which was authorizing

**Review by maxarvid — APPROVED** [review]
LGTM

**Pull request merged**